packages/mongodb-constants/src/expression-operators.ts

```diff
@@ -739,6 +739,34 @@ const EXPRESSION_OPERATORS = [
     meta: 'expr:array',
     version: '3.4.0',
   },
+  {
+    name: '$bitAnd',
+    value: '$bitAnd',
+    score: 1,
+    meta: 'expr:arith',
```

Comment:
Is their a more correct meta for these operators?

Reply:
I guess if it's a completely new type of operators, we can call them `expr:bit` or something. Tbh not even sure whether this is meaningfully used anywhere, except for just showing as-is in autocomplete

Reply:
Noice, on it 👍